added -i switch definition

ephestione opened the pull request.

Commit:
added -i switch definition

dolorosus commented:
Ok this defines the switch -i, but there is no function etc which will perform any action.

dolorosus committed:
Implementation test #1

ephestione commented:
uhm yes there is the -i switch in the code, look at row 453 ;)

dolorosus committed:
Update RaspiBackup.sh

Commit:
fixed loopback suffix and source suffix determination

Commit:
typo

dolorosus commented:
I know, but since now there was no useful implementation of the implicit needed rest (better nameing of LOOPBACK devices and so on). 
So up to now it wouldn't make sense to define a switch for an incomplete implemented function. Let's wait till the detemination of devices other than /dev/mmcblk is working.

Commit:
Merge branch 'Dynamic-Loopback-Dev-Names' into master

Commit:
cleared up merging mess!

I told you I'm a github newbie.
I thought I was pushing my changes into your branch, and instead I ended up pulling into my master your branch which I wanted to change in the first place! :rofl:

Commit:
typo

Commit:
Merge branch 'master' into master
